src/main/scala/dpla/ingestion3/entries/ingest/EnrichEntry.scala

```diff
@@ -87,22 +102,22 @@ object EnrichEntry {
       ExpressionEncoder.tuple(RowEncoder(model.sparkSchema), ExpressionEncoder())
 
     // Load the mapped records
-    val mappedRows: DataFrame = spark.read.avro(inputDir)
+    val mappedRows: DataFrame = spark.read.avro(dataIn)
```

Comment:
I like that you changed this variable name from `inputDir` to `dataIn`.

Reply:
Trying to normalize these var names across the project..